superset-frontend/packages/superset-ui-chart-controls/src/components/labelUtils.tsx

```diff
@@ -18,9 +18,41 @@
  */
 import React, { ReactNode } from 'react';
 
-import { t } from '@superset-ui/core';
+import { css, styled, t } from '@superset-ui/core';
 import { ColumnMeta, Metric } from '@superset-ui/chart-controls';
 
+const TooltipSectionWrapper = styled.div`
+  ${({ theme }) => css`
+    display: flex;
+    flex-direction: column;
+    font-size: ${theme.typography.sizes.s}px;
+    line-height: 1.2;
```

Comment:
Not relevant to this PR, but we should probably add a range of line heights to our theme at some point. CC @michael-s-molina